gateway-spi/src/main/java/org/apache/knox/gateway/services/security/AliasService.java

```diff
@@ -54,6 +54,8 @@ void generateAliasForCluster(String clusterName, String alias)
   char[] getPasswordFromAliasForGateway(String alias)
       throws AliasServiceException;
 
+  Map<String, char[]> getPasswordAliasMapForGateway() throws AliasServiceException;
```

Comment:
Why not simply getGatewayAliases(), more similar to other signatures in this interface? Why the need to cite the return type in the method signature?

Reply:
Done.